boards/arduino/opta/arduino_opta_stm32h747xx_m7.dts

```diff
@@ -23,22 +23,6 @@
 		zephyr,flash = &flash0;
 		zephyr,code-partition = &slot0_partition;
 	};
-
-	ethernet_phy_en: ethernet_phy_en {
-		compatible = "regulator-fixed";
-		regulator-name = "ethernet-phy-reset-release";
-		enable-gpios = <&gpioj 15 GPIO_ACTIVE_HIGH>;
-		regulator-boot-on;
-		status = "okay";
-	};
-
-	sdram2: sdram@d0000000 {
-		compatible = "zephyr,memory-region", "mmio-sram";
-		device_type = "memory";
-		reg = <0xd0000000 DT_SIZE_M(32)>;
-		zephyr,memory-region = "SDRAM2";
-		zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM) )>;
-	};
```

Comment:
This part of the change goes to the "device tree cleanup" commit. Also update that commit's description as it still refers to the SDRAM move.

Reply:
Sorry for the rebase, other projects I work with have the exact opposite policy (always rebase).

Anyway, I think now everything is as required.